build/NuSpecs/ProjectReunion-Nuget-Native.WinRt.props

```diff
@@ -6,20 +6,4 @@
     <_ProjectReunionFoundationPlatform Condition="'$(Platform)' != 'Win32'">$(Platform)</_ProjectReunionFoundationPlatform>
   </PropertyGroup>
```

Comment:
not needed

Reply:
But not harmful either. I was focused on deleting what's experimental but leaving anything inert as-is

@BenJKuhn which way do you want to go?